### PR TITLE
Ensure pod install is run if Pods directory is missing

### PR DESCRIPTION
**Pull request by pedropombeiro**
[comment]: # (Please replace ... with your information. Remove < and >)
[comment]: # (To auto-close issue on merge, please insert the related issue number after # i.e fixes #566)

### Summary

[comment]: # (Summarise the problem and how the pull request solves it)
This PR ensures `pod install` is run if you manually delete ios/Pods dir. It also makes `make release-ios` consistent with the other platforms by making it build from the command line.

Documentation change PR (review please): https://github.com/status-im/status.im/pull/xxx

### Testing notes

No manual testing needed, nothing in the software changed.

status: ready

### CODE REVIEW

File: src/status_im/ui/screens/chat/photos.cljs [hunk 1]
```diff
@@ -2,18 +2,21 @@
...
 
 (defn photo [photo-path {:keys [size
                                 accessibility-label]}]
-  [react/view {:style (style/photo-container size)}
-   [react/image {:source              (utils.image/source photo-path)
-                 :style               (style/photo size)
-                 :resize-mode         :cover
-                 :accessibility-label (or accessibility-label :chat-icon)}]
-   [react/view {:style (style/photo-border size)}]])
+  (let [identicon? (when photo-path (profile.db/base64-png? photo-path))]
```
Comment (vitvly) [inline src/status_im/ui/screens/chat/photos.cljs]: Why is this included here?

File: Makefile [hunk 2]
```diff
@@ -132,7 +132,7 @@ release-ios: watchman-clean ##@build build release for iOS release
 	$(MAKE) jsbundle-ios && \
 	echo "Build in XCode, see https://status.im/build_status/ for instructions" && \
 	scripts/copy-translations.sh && \
-	open ios/StatusIm.xcworkspace
+	xcodebuild -workspace ios/StatusIm.xcworkspace -scheme StatusIm -configuration Release -destination 'generic/platform=iOS' -UseModernBuildSystem=N clean archive
```
Comment (vitvly) [inline Makefile:133]: Just wondering (must have missed this earlier): why do we disable XCode's modern build system?

**Review by jakubgs — APPROVED** [review]
Not sure about the clojure stuff though.

File: src/status_im/ui/screens/chat/photos.cljs (hunk 1 above)
Comment (pedropombeiro) [reply src/status_im/ui/screens/chat/photos.cljs]: WHAT? No idea how it got here, I'll rebase the PR without this crap.

**Review by yenda — APPROVED** [review]

**Review by vitvly — APPROVED** [review]

File: Makefile (hunk 2 above)
Comment (pedropombeiro) [reply Makefile:133]: This has been there for quite a while, so I don't remember the details, but I believe there was a build error that wasn't easy to address without it.
